### DISCUSSION TITLE
Deprecated component library -> pink background

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#3206] [Help: Figma] [answered]
Hi team - wondering if anyone could give me a hot tip or two on how to get the pink background to show up in the side panel. 

I know you all have used this as a way to clearly indicate a deprecated component library, and I'd like to do that for the "[NEW] Evergreen Paste Components" as we switch over fully to "[NEW] Paste Components". 

I added the background to all component groups in the library, and somehow the Button one is the only one showing up after the merge (another issue), but even that one isn't showing up as pink in the side panel. 

Any help would be greatly appreciated, thank you!!

**Comment by loreina** [comment, marked as answer]
Hey @ceyerkes!

To add the background color, you'd have to add it to the component set (purple container with dashed border) as a hex code and not a token. 

We have a [design runbook](https://docs.google.com/document/d/1imyAsLLYt8Ng8j2w6P-ItgQQVhEpHyTrAkc2ZY0lXiM/edit#heading=h.exzg3bd4zx0p) with more detailed guidance on deprecating Figma components as well that may be useful for the Evergreen Paste Components library.

Shoot me a DM if you'd like to jam on this in real time!

**Reply by ceyerkes** [reply]
thank youuuuuu so much @loreina!
